### PR TITLE
fixing federation docs

### PR DESCRIPTION
**Pull request by agata-wit**
## Related Issue (if applicable)
Federation docs were all out of whack! Fixed it all now :)

## Types of changes

- [x] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [x] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/getting-started/key-concepts/graphql-federation.md [hunk 1]
```diff
@@ -7,36 +7,183 @@ menu:
...
+
+type User {
+  id: ID!
+  username: String!
+  reviews: [Review]
+}
+```
 
-### Creating a supergraph via the Dasboard UI
+### Creating a subgraph via the Dasboard UI
```
Comment (letzya) [inline tyk-docs/content/getting-started/key-concepts/graphql-federation.md:133]: I think this title is used twice in the same file

Comment (agata-wit) [reply tyk-docs/content/getting-started/key-concepts/graphql-federation.md:133]: I fixed this. There was actually a duplicate section in there, now all good.

**Comment by letzya** [comment]
Was reviewed by @agata-wit @pvormste

**Pull request merged**